gui/build.py

```diff
@@ -79,6 +80,20 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    env = dict(os.environ)
+    p = re.search('D:\\\\Python27\\\\Lib', env['PYTHONPATH'])
```

Comment:
This is why we have to use an environment var (or a command line flag). We need the Python system path for the cxfreeze command, but we can't discover it automatically it seems. So the idea is to define it with a default variable (I would say "C:\Python27" - as this is the most commn default I think) and let the user change it at runtime - by setting the environment var or using a command line flag.